hyperspectral/hyperspectral_calculation.py

```diff
@@ -54,10 +54,14 @@ def pixel2Geographic(jsonFileLocation, headerFileLocation, cameraOption):
         if "position x [m]" in master["gantry_system_variable_metadata"]:
             x_gantry_pos = float(master["gantry_system_variable_metadata"]["position x [m]"])
             y_gantry_pos = float(master["gantry_system_variable_metadata"]["position y [m]"])
-        else:
+
+        elif "Position x [m]" in master["gantry_system_variable_metadata"]:
             x_gantry_pos = float(master["gantry_system_variable_metadata"]["Position x [m]"])
             y_gantry_pos = float(master["gantry_system_variable_metadata"]["Position y [m]"])
 
+        else: # We notice that there are cases that no position data available
+            return [0], [0], ["0, 0"]*4 , ""
```

Comment:
would it make sense to have a missing data value here? otherwise, it is possible to confuse 0,0 with an actual position on the field.